gpt4all-backend/llmodel.cpp

```diff
@@ -51,10 +51,28 @@ Dlhandle *get_implementation(std::ifstream& f, const std::string& buildVariant)
 
 static
 bool requires_avxonly() {
-#ifdef __x86_64__
-    return !__builtin_cpu_supports("avx2") && !__builtin_cpu_supports("fma");
+#ifndef _MSC_VER
+    #ifdef __x86_64__
+        return !__builtin_cpu_supports("avx2") && !__builtin_cpu_supports("fma");
```

Comment:
I don't get the `fma` part here. Now the function behaves so that even if `__builtin_cpu_supports("avx2")` returns false, then FMA support still guarantees that AVX2 instructions are available?